popmon/pipeline/metrics.py

```diff
@@ -37,13 +38,10 @@
 
 def stability_metrics(
     hists,
+    settings: Settings,
     reference_type="self",
     reference=None,
     time_axis="",
-    window=10,
-    shift=1,
-    monitoring_rules=None,
-    pull_rules=None,
     features=None,
     **kwargs,
```

Comment:
Hey Simon, just one comment. 
We should probably take away the kwargs argument here.
This can also be fixed later, but right now it's a bit confusing.